vortex-array/src/array/bool/mod.rs

```diff
@@ -22,7 +22,6 @@ impl_encoding!("vortex.bool", ids::BOOL, Bool);
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct BoolMetadata {
     validity: ValidityMetadata,
-    length: usize,
     bit_offset: usize,
```

Comment:
this should be u8 since it's always <8

Reply:
addressed here https://github.com/spiraldb/vortex/pull/956